output/influxdb/config_test.go

```diff
@@ -51,3 +52,13 @@ func TestParseURL(t *testing.T) {
 		})
 	}
 }
+
+func TestGetConsolidatedConfig(t *testing.T) {
+	t.Parallel()
+	testdata := map[string]string{
+		"K6_INFLUXDB_PROXY": "http://localhost:3128",
+	}
+	config, err := GetConsolidatedConfig(nil, testdata, "")
+	require.NoError(t, err)
```

Comment:
```suggestion
	assert.Error(t, err)
```
We should also add a test case for the opposite case, checking that an error is returned as expected if an unparsable URL is passed.